### ISSUE TITLE
pytorch_scatter could maybe be faster for features aggregation

### ISSUE THREAD

**Issue by tchaton** [mys007/ecc#4]
https://github.com/rusty1s/pytorch_scatter

**Comment by mys007** [comment]
Sorry for the massive delay and thanks for making me aware of this framework. It's definitely worth a try, as well as looking into https://github.com/dmlc/dgl/ for a re-implementation....

**Comment by tchaton** [comment]
I think pytorch_geometric should also be an option.

**Comment by mys007** [comment]
Indeed, in fact, it seems that they've even implemented ECC themselves: https://github.com/rusty1s/pytorch_geometric/blob/master/torch_geometric/nn/conv/nn_conv.py !
